src/Solver/FreeSurfaceIntegrator.cpp

```diff
@@ -162,9 +162,12 @@ void seissol::solver::FreeSurfaceIntegrator::initializeProjectionMatrices(unsign
   // Sub triangles
   triRefiner.refine(maxRefinementDepth);
 
+  const auto projectionMatrixNCols = tensor::subTriangleProjection::Shape[tensor::subTriangleProjection::index(maxRefinementDepth)][1];
```

Comment:
`constexpr`?

Reply:
Ok; thanks, missed that one.

Reply:
Tried it briefly but compiler said NO :-D